daffodil-test/src/test/resources/org/apache/daffodil/section05/simple_types/SimpleTypes.tdml

```diff
@@ -51,7 +51,11 @@
     <xs:element name="int02" type="xs:int" dfdl:lengthKind="explicit"
       dfdl:length="11" />
     <xs:element name="int03" type="xs:int" dfdl:representation="binary"
-      dfdl:binaryNumberRep="binary" />
+      dfdl:binaryNumberRep="binary">
+      <xs:annotation>
+        <xs:appinfo source="http://www.ogf.org/dfdl/dfdl-1.0/" />
+      </xs:annotation>
+    </xs:element>
```

Comment:
I think you meant change representation to text, when we do, we get an error about representation text not being allowed for lengthkind implicit. Changing the lengthKind to explicit with length 4 bytes, we get the parse error "Unable to parse xs:int from text: (NUL)(NUL)(NUL)%". No warning either ways.

I did find this in the spec, but maybe the warning is optional?
     
```
Property is not applicable because of another DFDL property setting.
    Warning (optional). Example is dfdl:binaryNumberRep when dfdl:representation is text.
```

Reply:
The "property not applicable" is a type of warning that we do generate (though our warning is a little less helpful, just saying a property is ignored and leave it up to the user to figure out why).

It's possible the warning is being hidden because of the parse error/test failure? If you change documentPart type="text" and give it a 4-digit text string so the test passes do you see the warning?

I've tested this simple schema and I do get the warning:

```xml
  <element name="foo" type="xs:string" dfdl:length="1" dfdl:lengthKind="explicit" dfdl:binaryNumberRep="binary" />

```

Note that it looks like this test was created a long time ago just to test that we could create and expect warnings back when that was a new thing. I think it's well established that that works now. We also have tests specifically about ignore warnings (e.g. `ignoredPropertiesWarning` in PropertySyntax.tdml. So this test may not even be needed anymore and could maybe be removed. Unless there is value in having a test explicitly checking for binaryNumberRep being ignored. I don't think that's a useless test, but I also don't think it's a super important test to have.

Reply:
Ah yes, I get the warning now. Many thanks